tests/kafkatest/tests/verifiable_consumer_test.py

```diff
@@ -56,7 +55,7 @@ def min_cluster_size(self):
     def setup_consumer(self, topic, static_membership=False, enable_autocommit=False,
                        assignment_strategy="org.apache.kafka.clients.consumer.RangeAssignor", group_remote_assignor="range", **kwargs):
         return VerifiableConsumer(self.test_context, self.num_consumers, self.kafka,
-                                  topic, self.group_id, static_membership=static_membership, session_timeout_sec=self.session_timeout_sec,
```

Comment:
`session_timeout_sec` is used by other end-to-end tests, so we need to perform some refactoring for those test cases.

**Replace Dynamic Timeouts with Constants**
For example, change timeout_sec=self.session_timeout_sec + 5 to timeout_sec=60.

**Remove Use Cases of Increasing session_timeout_sec**
These cases typically increase session_timeout_sec from 45 seconds to 60 seconds. This adjustment may be unnecessary if the tests can run stably without modifying the timeout.

Reply:
I tested e2e client again. `TC_PATHS="tests/kafkatest/tests/client" bash tests/docker/run_tests.sh` and only one test failed.

```
test_id:    kafkatest.tests.client.consumer_test.OffsetValidationTest.test_broker_rolling_bounce.metadata_quorum=ISOLATED_KRAFT.use_new_coordinator=False
status:     FAIL
run time:   3 minutes 23.970 seconds


    AssertionError('Broker rolling bounce caused 7 unexpected group rebalances')
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/ducktape/tests/runner_client.py", line 351, in _do_run
    data = self.run_test()
  File "/usr/local/lib/python3.7/dist-packages/ducktape/tests/runner_client.py", line 411, in run_test
    return self.test_context.function(self.test)
  File "/usr/local/lib/python3.7/dist-packages/ducktape/mark/_mark.py", line 438, in wrapper
    return functools.partial(f, *args, **kwargs)(*w_args, **w_kwargs)
  File "/opt/kafka-dev/tests/kafkatest/tests/client/consumer_test.py", line 121, in test_broker_rolling_bounce
    "Broker rolling bounce caused %d unexpected group rebalances" % unexpected_rebalances
AssertionError: Broker rolling bounce caused 7 unexpected group rebalances
```

And looks like it was tracked by https://issues.apache.org/jira/browse/KAFKA-18194